packages/ui/src/molecules/QuantitySelector/QuantitySelector.test.tsx

```diff
@@ -0,0 +1,92 @@
+import { render } from '@testing-library/react'
+import React, { useState } from 'react'
+import userEvent from '@testing-library/user-event'
+import { axe } from 'jest-axe'
+
+import QuantitySelector from '.'
+
+const QuantitySelectorTest = () => {
+  const [quantity, setQuantity] = useState(1)
+
+  return (
+    <QuantitySelector
+      name="quantity-selector"
+      quantity={quantity}
+      leftButtonProps={{
+        icon: <span>-</span>,
+        onClick: () => {
+          setQuantity((curr) => curr - 1)
+        },
+      }}
+      rightButtonProps={{
+        icon: <span>+</span>,
+        onClick: () => {
+          setQuantity((curr) => curr + 1)
+        },
+      }}
+      inputProps={{ readOnly: true }}
+    />
+  )
+}
+
+describe('QuantitySelector', () => {
+  it('should render QuantitySelector with its children', () => {
+    const { getByTestId } = render(
+      <QuantitySelector
+        name="quantity-selector"
+        quantity={1}
+        leftButtonProps={{ icon: <span>-</span> }}
+        rightButtonProps={{ icon: <span>+</span> }}
+        inputProps={{ readOnly: true }}
+      />
+    )
+
+    expect(getByTestId('store-quantity-selector')).toBeInTheDocument()
+  })
+
+  it('should render QuantitySelector with the correct quantity', () => {
+    const { getByLabelText } = render(
+      <QuantitySelector
+        name="quantity-selector"
+        quantity={123}
+        leftButtonProps={{ icon: <span>-</span> }}
+        rightButtonProps={{ icon: <span>+</span> }}
+        inputProps={{ readOnly: true }}
+      />
+    )
+
+    expect(getByLabelText('Quantity')).toHaveValue('123')
+  })
+
+  it('Should update correctly according to event changes', () => {
+    const { getByLabelText } = render(<QuantitySelectorTest />)
+
+    expect(getByLabelText('Quantity')).toHaveValue('1')
+
+    const rightButton = getByLabelText('Increment Quantity')
+    const leftButton = getByLabelText('Decrement Quantity')
+
+    userEvent.click(rightButton)
+    expect(getByLabelText('Quantity')).toHaveValue('2')
+    userEvent.click(leftButton)
+    expect(getByLabelText('Quantity')).toHaveValue('1')
+  })
+
+  describe('QuantitySelector Accessibility Tests', () => {
+    it('Should pass the Accessibility test with AXE', async () => {
+      render(
+        <main>
+          <QuantitySelector
+            name="quantity-selector"
+            quantity={1}
+            leftButtonProps={{ icon: <span>-</span> }}
+            rightButtonProps={{ icon: <span>+</span> }}
+            inputProps={{ readOnly: true }}
+          />
+        </main>
+      )
+
+      expect(await axe(document.body)).toHaveNoViolations()
```

Comment:
I wonder if it's worth updating this branch with `master` to assert for the recent addition of [`toHaveNoIncompletes`](https://github.com/vtex/faststore/pull/1054). 

I noticed there's 1 warning you can currently see on [Storybook's Accessibility tab](https://deploy-preview-1083--faststoreui.netlify.app/?path=/story/molecules-quantityselector--quantity-selector):

![Screen Shot 2021-12-20 at 12 54 40](https://user-images.githubusercontent.com/381395/146795645-0b1b4019-3e81-4fd6-8595-1659e71a84ac.png)

Reply:
Done at https://github.com/vtex/faststore/pull/1083/commits/162fe129d1340692e2b43f335695b1ddb28e81f4 and https://github.com/vtex/faststore/pull/1083/commits/45ff677aa1e27694cb5d0ea60abe798ec3cd65b8